ci/build_container/build_recipes/gperftools.sh

```diff
@@ -6,13 +6,16 @@ if [[ "${OS}" == "Windows_NT" ]]; then
   exit 0
 fi
 
-VERSION=2.7
-SHA256=1ee8c8699a0eff6b6a203e59b43330536b22bbcbe6448f54c7091e5efb0763c9
+VERSION=fc00474ddc21fff618fc3f009b46590e241e425e
+SHA256=a2b6da3562addd0ffba2cc7731867033cd0f548888053a22dc358226ad005912
 
-curl https://github.com/gperftools/gperftools/releases/download/gperftools-"$VERSION"/gperftools-"$VERSION".tar.gz -sLo gperftools-"$VERSION".tar.gz \
+curl https://api.github.com/repos/gperftools/gperftools/tarball/"$VERSION" -sLo gperftools-"$VERSION".tar.gz \
```

Comment:
`https://github.com/gperftools/gperftools/archive/${VERSION}.tar.gz`?

Reply:
Thanks! For some reason a tarball downloaded via a similar method was not working before, but it appears to work fine now.